src/evaluations/run_evaluation.py

```diff
@@ -14,12 +14,13 @@
 r"""A commandline tool to run evaluations of cardinality estimators.
 
 To run this evaluation, in command line, type:
-python run_evaluation.py \
---evaluation_out_dir="evaluation_output" \
---analysis_out_dir="analysis_output" \
---evaluation_config="smoke_test" \
---sketch_estimator_configs="vector_of_counts-4096-ln3-sequential" \
---evaluation_run_name="simple_run"
+  wfa-run-evaluation \
```

Comment:
The other way will still be used by those who develop.

I think maybe you want this documentation in main.py?

With perhaps a note here to look at main.py for those who have the environment installed?

Reply:
I actually intend for this to be used by developers, too. Is there a reason that devs should use the other way? 

Encouraging devs to use this way will ensure that the version of run_evaluation.py they are using always matches up with the version of the package they have installed. For example, when switching branches, running `python run_evaluation.py` without first running `python setup.py install` will run the run_evaluation.py script of the current branch while running the code of the previous branch which could cause nontrivial, hard-to-debug behaviors. Running `wfa-run-evaluation`, on the other hand, will always use the run_evaluation.py that is installed in the current environment, ensuring consistent behavior while developing.

Reply:
Makes sense. Thanks for the explanation.